### DISCUSSION TITLE
Suggestions for New Features

### DISCUSSION THREAD

**Discussion by aryangoel24** [CSC207-2022F-UofT/course-project-budgetbites#66] [Ideas]
Hey guys, let this be a discussion board that will be open until the project deadline, for everyone to share and review suggestions about new features. For big features, we can create new discussion boards with the poll option to decide whether or not to implement the feature or not.

**Comment by Aviral-03** [comment]
We need to submit the project by Monday. Please ensure that you create a pull request as soon as possible with your tests so that we can finally work on clean architecture.
